src/index.ts

```diff
@@ -1,8 +1,52 @@
 import * as cookie from 'cookie'
 import * as next from 'next'
+import * as setCookieParser from 'set-cookie-parser'
+import { Cookie } from 'set-cookie-parser'
+import { CookieSerializeOptions } from 'cookie'
 
 const isBrowser = () => typeof window !== 'undefined'
 
+/**
+ * Compare the cookie and return true if the cookies has equivalent
+ * options and the cookies would be overwritten in the browser storage.
+ *
+ * @param a first Cookie for comparision
+ * @param b second Cookie for comparision
+ */
+function areCookiesEqual(a: Cookie, b: Cookie) {
```

Comment:
I believe the name of this function should be the exact opposite. Am I missing something here?

Reply:
You are right. The function does really the exact opposite. I missed this during a refactoring. Will fix it tomorrow.